libs/langchain/langchain/vectorstores/neo4j_vector.py

```diff
@@ -328,6 +407,17 @@ def __from(
                 f"Vector index dimension: {embedding_dimension}"
             )
 
+        if search_type == SearchType.HYBRID:
```

Comment:
should this also be done in init?

Reply:
Not really, because here we create a keyword index if missing, but when using `from_existing_index` method, we don't want to create a missing index but rather raise an error:

https://github.com/tomasonjo/langchain/blob/neo4jhybrid/libs/langchain/langchain/vectorstores/neo4j_vector.py#L743